docs/quickstart.md

```diff
@@ -2,198 +2,441 @@
 
 >>>  # this will still be a pynapple time series
 >>> print(type(firing_rate))  # shape (num_samples, )
 <class 'pynapple.core.time_series.Tsd'>
+
 ```
 
 Let's see how you can greatly streamline your analysis pipeline by integrating `pynapple` and NeMoS.
 
+
 !!! note
     You can download this dataset by clicking [here](https://www.dropbox.com/s/su4oaje57g3kit9/A2929-200711.zip?dl=1).
 
+
 ```python
-import nemos as nmo
-import numpy as np
-import pynapple as nap
 
-data = nap.load_file("A2929-200711.nwb")
+>>> import nemos as nmo
+>>> import pynapple as nap
+
+>>> path = nmo.fetch.fetch_data("A2929-200711.nwb")
+>>> data = nap.load_file(path)
+
+>>> # load spikes and head direction
+>>> spikes = data["units"]
+>>> head_dir = data["ry"]
 
-spikes = data["units"]
-head_dir = data["ry"]
+>>> # restrict and bin
+>>> counts = spikes[6].count(0.01, ep=head_dir.time_support)
 
-counts = spikes[6].count(0.01, ep=head_dir.time_support)  # restrict and bin
-upsampled_head_dir = head_dir.bin_average(0.01)  # up-sample head direction
+>>> # down-sample head direction
+>>> upsampled_head_dir = head_dir.bin_average(0.01)  
 
-# create your features
-X = nmo.basis.CyclicBSplineBasis(10).compute_features(upsampled_head_dir)
+>>> # create your features
+>>> X = nmo.basis.CyclicBSplineBasis(10).compute_features(upsampled_head_dir)
+
+>>> # add a neuron axis and fit model
+>>> model = nmo.glm.GLM().fit(X, counts) 
 
-# add a neuron axis and fit model
-model = nmo.glm.GLM().fit(X, counts) 
 ```
 
+
 Finally, let's compare the tuning curves
 
+
 ```python
-import numpy as np
-import matplotlib.pyplot as plt
 
-raw_tuning = nap.compute_1d_tuning_curves(spikes, head_dir, nb_bins=100)[6]
-model_tuning = nap.compute_1d_tuning_curves_continuous(
-    model.predict(X)[:, np.newaxis] * X.rate,  # scale by the sampling rate
-    head_dir,
-    nb_bins=100
-)[0]
+>>> import numpy as np
+>>> import matplotlib.pyplot as plt
+
+>>> # tuning curves
+>>> raw_tuning = nap.compute_1d_tuning_curves(spikes, head_dir, nb_bins=100)[6]
+
+>>> # model based tuning curve
+>>> model_tuning = nap.compute_1d_tuning_curves_continuous(
+...     model.predict(X)[:, np.newaxis] * X.rate,  # scale by the sampling rate
+...     head_dir,
+...     nb_bins=100
+...  )[0]
+
+
+>>> # plot results
+>>> sub = plt.subplot(111, projection="polar")
+>>> plt1 = plt.plot(raw_tuning.index, raw_tuning.values, label="raw")
+>>> plt2 = plt.plot(model_tuning.index, model_tuning.values, label="glm")
+>>> legend = plt.yticks([])
+>>> xlab = plt.xlabel("heading angle")
 
-# plot results
-plt.subplot(111, projection="polar")
-plt.plot(raw_tuning.index, raw_tuning.values, label="raw")
-plt.plot(model_tuning.index, model_tuning.values, label="glm")
-plt.legend()
-plt.yticks([])
-plt.xlabel("heading angle")
-plt.show()
 ```
 
-![Alt text](head_dir_tuning.jpg)
 
-### Compatibility with `scikit-learn`
+<img src="../head_dir_tuning.jpg" width="40%">
```

Comment:
we can't generate this image as part of the docs build?

Reply:
this won't run without any extra machinery, only gallery ".py" files are run.